site/_docs/howto.md

```diff
@@ -457,12 +457,27 @@ particular release managers.
 
 ## Merging pull requests (for Calcite committers)
 
```

Comment:
Add "These are instructions for a Calcite committer who has reviewed a pull request from a contributor, found it satisfactory, and is about to merge it to master. Usually the contributor is not a committer (otherwise they would be committing it themselves, after you gave approval in a review)."